src/redux/session/thunks.js

```diff
@@ -14,8 +14,11 @@ export const loginThunk = (mnemonic) => (dispatch) => {
     const privateKey = getPrivateKeyByMnemonic(mnemonic)
     const bitcoinAddress = getAddress(privateKey)
     const ethAddress = getAddressByMnemonic(mnemonic)
-    dispatch(bitcoinCreateWallet(bitcoinAddress))
     dispatch(ethereumCreateWallet(ethAddress))
+    dispatch(ethereumCreateWallet(`0x${bitcoinAddress}`))
```

Comment:
Could you please explain login at lines 17-21. Can't figure it out...

Reply:
They were for testing purposes